src/test/java/org/salesforce/steps/featureSteps.java

```diff
@@ -36,9 +36,9 @@ public void iLoginToSalesforceSiteAsAnAdminUser() {
                 EnvConfig.getInstance().getPassword());
     }
 
-    @When("I navigate to {string} page in mode {string}")
-    public void iNavigateToPageInMode(String arg0, String arg1) {
-        pageTransporter.navigateToWorkTypePageLightningDirect();
+    @When("I navigate to the {string} page")
+    public void iNavigateToThePage(String arg0) {
```

Comment:
Change name of parameter input for something more descriptive

Reply:
ok I will change the name of parameter